Automattic/wp-calypso
# Limit Global Styles: Update the Pre-Save Notice

#### Proposed Changes
Fixes: https://github.com/Automattic/dotcom-forge/issues/1221

* Changed the copy of the pre-save notice.
* Stop showing the reset link in the notice's message.
* Refactor the observer that inserts the notice in the pre-save panel.

| Before | After |
| - | - |
| ![image](https://user-images.githubusercontent.com/2070010/204907115-6be31984-bac4-4b82-ae3d-87a9b0a2e7a3.png) | <img alt="Screenshot 2022-11-30 at 20 58 57" src="https://user-images.githubusercontent.com/2070010/204907141-cfebe7f0-b12d-4de4-99b1-22af11d5865d.png"> |

#### Testing Instructions



- Setup:
  - Apply these changes to your sandbox: `install-plugin.sh editing-toolkit update/gs-pre-save-panel-notice`
  - Go to https://wordpress.com/start and create a new free site
  - Add the `wpcom-limit-global-styles` blog sticker to the new site: `add_blog_sticker( 'wpcom-limit-global-styles', 'test', <YOUR_USERNAME>, <YOUR_BLOG_ID>)`.
  - Sandbox the new site.
- Open the Site Editor.
- Open the Global Styles sidebar.
- ⚠️ _(check for regression)_ Ensure the paid Global Styles modal appears.
- Do some changes in the canvas (not on Global Styles just yet) and click "Save".
- ⚠️ Ensure the Global Styles notice does not appear.
- Do some changes to Global Styles and click "Save" again.
- ⚠️ Ensure the Global Styles notice appears and is appended to the correct entity.
- Try playing around with the Global Styles customizations and toggling the pre-save panel.
- ⚠️ Ensure the Global Styles notice behaves as expected (appears and disappears when it should).

#### Pre-merge Checklist





Related to #
